technical_docs/Claims.md

```diff
@@ -1,6 +1,6 @@
 
-SciToken Claims Language
-========================
+SciToken Claims and Scopes Language
+====================================
 
 Each SciToken has one or more claim associated with it.  These claims are typically used to indicate the authorizations the SciToken bearer may have -- but other use cases exist (such as adding monitoring or auditing information into the token chain).
 
```

Comment:
I think it's more correct to say that each deployment of SciTokens can add domain-specific claims. Since the token is signed by its issuer, the downstream user may not extend it.